source/developer/fx-guidelines.rst

```diff
@@ -73,15 +73,29 @@ We’re using `BEM methodology <https://en.bem.info/method/>`_  and would advise
 
 -----
 
-Minimum Display Requirements
+Screen and Content Dimensions
 ****************************
 
-Desktop
--------
+Web/Desktop App
+-------------------
 
-- **Minimum screen resolution:** 1280 x 800
+The webapp supports two main modes of displaying the app: desktop view and mobile view. The app is primarily used in desktop view, but when the screen is 768 pixels or less wide, it switches to mobile view. The mobile view is also used in mobile browsers and the Classic App.
 
-- **Color Depth:** 16.7M colors
+For desktop view:
+
+ - The target screen sizes are 1280 x 800 or 1920 x 1080. Note that the space available for content will be smaller than this due to the the frame of the window and system-specific components like the taskbar on Windows. For example, the desktop app on Windows 10 only has a usable height of 717 pixels when the tab bar is visible at 1280 x 800 resolution.
+ - The minimum content size is 769 x 600. When the width is 768 or less, the app switches to mobile view. When the height is below 600, UI elements may become misplaced.
```

Comment:
So it's the width that determines the view right? Since we say "When the height is below 600, UI elements may become misplaced." but the condition would be that the width is also above 768? Can we clarify?

Reply:
`The minimum content size is 769 x 600`.
We should change that to `768`.